crates/rolldown_resolver/src/resolver.rs

```diff
@@ -108,9 +108,11 @@ impl<F: FileSystem + Clone> Resolver<F> {
 
     let resolve_options_with_default_conditions = OxcResolverOptions {
       cwd: Some(cwd.clone()),
-      tsconfig: tsconfig.map(|tsconfig| oxc_resolver::TsconfigOptions {
-        config_file: tsconfig,
-        references: oxc_resolver::TsconfigReferences::Disabled,
+      tsconfig: tsconfig.map(|tsconfig| {
+        oxc_resolver::TsconfigDiscovery::Manual(oxc_resolver::TsconfigOptions {
```

Comment:
Core team need to follow up with a PR to enable automatic tsconfig discovery.

Reply:
@sapphi-red cc

Reply:
I think we should keep this disabled for rolldown itself until we have the `include` & `exclude` feature implemented so that we don't have to introduce a breaking change twice.